sdcm/cluster.py

```diff
@@ -1606,7 +1606,7 @@ def datacenter_setup(self, datacenters):
         cmd = cmd.format(datacenters[self.dc_idx], dc_suffix)
         self.remoter.run(cmd)
 
-    def patch_scylla_yaml(self, yaml_file=SCYLLA_YAML_PATH, **kwargs):
+    def patch_scylla_yaml(self, yaml_file=SCYLLA_YAML_PATH, seed_address=None, **kwargs):
```

Comment:
don't add `seed_address`. use `seed_provider` and don't put the whole structure of it in this function 
or this function would become `node_config` all over again

Reply:
fixed